modules/op-release-notes-1-14.adoc

```diff
@@ -25,6 +25,8 @@ In addition to fixes and stability improvements, the following sections highligh
 
 * With this update, object parameters and array results are enabled by default. You do not need to set any flags to use them.
 
+* With this update, `PipelineRun` details page in the *Developer* or *Administrator* perspective of the web console provides a visual representation of identified vulnerabilities, categorized by the severity (critical, high, medium, and low). Additionally, it provides an option to download or view Software Bill of Materials (SBOMs) for enhanced transparency and control within your supply chain.
```

Comment:
Please move this paragraph into the Web console section of the RN. Also, please add the information about what must be installed to have this feature; it is not available in a default installation.